airtable/plugin.py

```diff
@@ -159,19 +168,26 @@ def airtable_record_list(
     base_id: AirtableBaseID,
     table_id: AirtableTableID,
     include_fields: Optional[set[str]] = None,
+    view_id: Optional[AirtableViewID] = None,
 ) -> list[AirtableRecord]:
     """
-    Return results of an Airtable `list records` API call with the option of choosing
-    which fields to include in the returned records.  The include_fields fields are
-    required, so populate it to ensure that the returned records have the fields you
-    will access by name.
-
-    If `include_fields` is not `None`, only the fields specified in `include_fields`
-    will be included.  If `include_fields` is `None`, all fields are included.
+    Returns the results of an Airtable `list records` API call, allowing selection of
+    specific fields to include in the returned records. Fields present in the schema
+    but not assigned a value will not be returned.
```

Comment:
Does the part about the fields actually make a difference?  It seems only relevant to code gen, and the rest of the comment below addresses it.